src/entity-generator/entity-generator.cpp

```diff
@@ -117,20 +145,30 @@ rapidjson::Value getJsonPath(std::array<double, 3> location,
   return path;
 }
 
+std::array<double, 3> getGridLocation(int index, int entityCount) {
+  int side = ceil(sqrt(static_cast<double>(entityCount)));
+  int row = index / side;
+  int col = index % side;
+  return {{static_cast<double>(col) * options.stepSize * 3,
+           static_cast<double>(row) * options.stepSize * 3, 0}};
+}
+
 void initializeEntities() {
   for (int i = 0; i < options.entityCount; ++i) {
     Entity newEntity;
     newEntity.id = str(boost::format("live-test-%1%") % i);
-    if (options.centerStart) {
+    if (options.testPattern) {
+      newEntity.location = getGridLocation(i, options.entityCount);
+    } else if (options.centerStart) {
       newEntity.location = CENTER_OF_US;
     } else {
       newEntity.location = {{start_lon(), start_lat(), 0.}};
     }
+    newEntity.initialLocation = newEntity.location;
     if (options.live) {
       newEntity.timestamp = NowGMT();
     } else {
-      // newEntity.timestamp = JAN_01_1996_GMT;
-      newEntity.timestamp = NowGMT() - ONE_WEEK;
```

Comment:
@jrmConduce The option to start at time now still exists, if you specify the `--live` flag.  The old behavior default was to set the start time to now minus one week.  That seemed arbitrary.  A good option for setting start time to now might be to write a python script that retrieves "now" from the OS and passes the value as the start time to entity-generator.